common/src/main/java/com/wynntils/core/features/overlays/Edge.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.features.overlays;
+
+import com.wynntils.utils.objects.Pair;
+import java.util.Map;
+import net.minecraft.world.phys.Vec2;
+
+public enum Edge {
+    Top(Corner.TopLeft, Corner.TopRight),
+    Left(Corner.TopLeft, Corner.BottomLeft),
+    Right(Corner.TopRight, Corner.BottomRight),
+    Bottom(Corner.BottomLeft, Corner.BottomRight);
```

Comment:
I'm not sure I should even write this, since it doesn't matter too much, but the relationship of Edge and Corner feels kind of backwards. At least if you compare with how things are named in English. A "Top Left" corner sounds like it combines a "Top" and "Left" edge. That is, edge feels more atomic, and corners are defined given which edges meet, instead of having corners atomic and edges defined by which corners they combine.

Feel free to ignore this if you think it creates too much work to redesign this.